app/services/email_sender_service/email_sender.rb

```diff
@@ -0,0 +1,45 @@
+module EmailSenderService
+  class EmailSender
+    def call(address:, subject:, body:)
```

Comment:
Might be nice to have a log entry here that explains if it's overriden or not.

E.g. 
```
overriden = " (overriden to #{email_address_override}) " if email_address_overide
logger.info(%(Sending email to #{address}#{overriden} with subject: #{subject}")
```

and then adjust the pseudo one accordingly.

Reply:
I've added this in here https://github.com/alphagov/email-alert-api/pull/215/files#diff-51c7eba122752bad14917f721d539715R8 for now. I haven't updated the pseudo class yet because I've yet to decide how to implement it nicely. Either I start passing in extra params when calling the Pseudo method (and then have to change the interface to the Notify class as well, or have a `switch` / `if` condition in here) or I just make the Pseudo class a complete noop and implement all the logging in here.....or something else entirely. haven't figured it out yet.